packages/web-app-preview/src/App.vue

```diff
@@ -525,6 +520,43 @@ export default defineComponent({
     imageRotateRight() {
       this.currentImageRotation =
         this.currentImageRotation === 270 ? 0 : this.currentImageRotation + 90
+    },
+    isFileTypeImage(file) {
+      return !this.isFileTypeAudio(file) && !this.isFileTypeVideo(file)
+    },
+    isFileTypeAudio(file) {
+      return file.mimeType.toLowerCase().startsWith('audio')
+    },
+
+    isFileTypeVideo(file) {
+      return file.mimeType.toLowerCase().startsWith('video')
+    },
+    preloadImages() {
+      for (const file of this.filteredFiles) {
+        if (!this.isFileTypeImage(file)) {
+          continue
+        }
+        loadPreview({
+          resource: file,
+          isPublic: this.isPublicLinkContext,
+          server: configurationManager.serverUrl,
+          userId: this.user.id,
+          token: this.accessToken,
+          dimensions: [this.thumbDimensions, this.thumbDimensions] as [number, number]
+        }).then((mediaUrl) => {
+          this.cachedFiles.push({
+            id: file.id,
+            name: file.name,
+            url: mediaUrl,
+            ext: file.extension,
+            mimeType: file.mimeType,
+            isImage: true,
+            isVideo: false,
+            isAudio: false
+          })
+          console.log(this.cachedFiles)
+        })
+      }
```

Comment:
I know this is still in draft state, but just to make sure.. you're planning on loading previews for the next/previous few images not all in a folder, right? :trollface: 😅

Reply:
on it lol